tests/fv3_conf/cpld_fv3_mom6_cice_ww3_bmark_rt_run.IN

```diff
@@ -44,12 +48,44 @@ cp    @[RTPWD]/CICE_FIX/@[OCNRES]/mesh.mx@[OCNRES].nc .
 cp    @[RTPWD]/WW3_input_data/mod_def.* .
 cp    @[RTPWD]/WW3_input_data/ww3_multi.inp .
 
-# ICs
-cp    @[RTPWD]/${FV3_IC}/sfc_data*.nc ./INPUT
-cp    @[RTPWD]/${FV3_IC}/gfs_data*.nc ./INPUT
-cp    @[RTPWD]/${MOM6_IC}/MOM*.nc ./INPUT
+# No restart
+if [ $WARM_START = .F. ]; then
+  # ICs
+  if [ $FRAC_GRID = .F. ]; then
+  # cp  @[RTPWD]/${FV3_DIR}/INPUT/sfc_data*.nc ./INPUT
+  # cp  @[RTPWD]/${FV3_DIR}/INPUT/gfs_data*.nc ./INPUT
+   cp  @[RTPWD]/${FV3_IC}/sfc_data*.nc ./INPUT
+   cp  @[RTPWD]/${FV3_IC}/gfs_data*.nc ./INPUT
+  else
+   cp  @[RTPWD]/FV3_input_frac/@[ATMRES].mx@[OCNRES]_frac/sfc_data*.nc ./INPUT
+   cp  @[RTPWD]/FV3_input_frac/@[ATMRES].mx@[OCNRES]_frac/gfs_data*.nc ./INPUT
+  fi
+ # cp  @[RTPWD]/MOM6_IC/MOM*.nc ./INPUT
+ # cp  @[RTPWD]/CICE_IC/@[OCNRES]/cice_model_@[ICERES].res_2016100300.nc ./cice_model.res.nc
+  cp  @[RTPWD]/${MOM6_IC}/MOM*.nc ./INPUT
 # the BM ICs are still named cice5 and need to remain so until P5.0 is completed
-cp    @[RTPWD]/${CICE_IC}/cice5_model_@[ICERES].*.nc ./cice_model.res.nc
+  cp  @[RTPWD]/${CICE_IC}/cice5_model_@[ICERES].*.nc ./cice_model.res.nc
+
+# Restart
+else
+  # Restart files
```

Comment:
Shan, Have you tested the restart reproducibility with fractional test (FV3-MOM6-CICE6)?